### PR TITLE
Tenant Widget

### PR DESCRIPTION
**Pull request by rishsingh07**
Fixes #

Changes in this pull request:
-
-
-

@

### CODE REVIEW

**Comment by norbertschuler** [comment]
image

**Comment by sagararyal** [comment]
Few Notes:

Available props for tenant counter are:

tenantKey
target
locale
primaryColor
circleBGColor

eg:

<tree-tenantcounter tenantKey='ten_xy123' target="100" locale="en"></tree-tenantcounter>

We do not need dark theme.

File: src/TreeTenantCounter/App.svelte [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+            break;
+        default:
+            language = enLocale;
+            break;
+    }
+
+    let userpofiledata;
+    const fetchProfileData = (async () => {
+        const response = await fetch(`${__myapp.env.API_URL}/tenantScore`,{headers: {
+        'tenant-key': "ten_I9TW3ncG"}});
```
Comment (sagararyal) [inline src/TreeTenantCounter/App.svelte:42]: this should be passed in props

File: src/TreeTenantCounter/App.svelte [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+    }
+
+    .logoPlanet > img {
+        height: 52px;
+        width: 52px;
+    }
+
+    .logoPlanet > svg {
+        border-radius: 50%;
+    }
```
Comment (sagararyal) [inline src/TreeTenantCounter/App.svelte:271]: unused css

File: src/TreeTenantCounter/App.svelte [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+
+    .infoIcon > .infoText {
+        display: none;
+    }
+    .infoIcon:hover {
+        cursor: pointer;
+    }
+    .infoIcon:hover > .infoText {
+        display: block;
+    }
```
Comment (sagararyal) [inline src/TreeTenantCounter/App.svelte:316]: unused css

**Comment by harshvitra** [comment]
Width of the containers are not stable for tree tenant leaderboard

**Review by harshvitra — APPROVED** [review]

**Pull request merged**